<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MVCClass</class>
 <widget class="QMainWindow" name="MVCClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MVC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOfOriginalImage">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOfColor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOfThreshold">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Threshold:</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderOfThreshold">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>22</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="valueOfColor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="valueOfThreshold">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOfProcessImage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mvc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sliderOfThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MVCClass</receiver>
   <slot>on_sliderOfThreshold_valueChaned()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderOfThreshold</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MVCClass</receiver>
   <slot>on_sliderOfThreshold_sliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_openButton_clicked()</slot>
  <slot>on_sliderOfThreshold_valueChaned()</slot>
  <slot>on_processButton_clicked()</slot>
  <slot>on_sliderOfThreshold_sliderMoved()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
